models/Invitation.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const InvitationSchema = new Schema({
+  status: {
+    type: String,
+    default: "pending",
+  },
+  referrer: {
+    type: Schema.Types.ObjectId,
+    ref: "users", //references the users model
+  },
+  toEmail: {
+    type: String,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+});
```

Comment:
We'll also need a way to know which invitation is linked to a given referral url. We could add a field for this, or perhaps just use the _id

Reply:
Was planning on using the unique _id field generated!